app/assets/javascripts/app/views/help_view.js

```diff
@@ -133,6 +140,23 @@ app.views.Help = app.views.StaticContentView.extend({
     this.$('#faq').append(help_section.render().el);
   },
 
+  /**
+   * Returns The section title whose data-section property equals the given query
+   * Returns null if nothing found
+   * @param dataValue Value for the data-section to find
+   * @returns {jQuery}
+   */
+  findSection: function(dataValue){
```

Comment:
Well I started to call them section, bu as you can see, there's already a variable `section` in the `render` function, so I do what I can. But I can still refactor to call it `findTopic` if it really bothers you.